py/embedded-server/deephaven_server/start_jvm.py

```diff
@@ -46,14 +46,22 @@ def _jars():
 
 # Provide a util func to start the JVM, will use its own defaults if none are offered
 def start_jvm(
-        jvm_args = DEFAULT_JVM_ARGS,
-        jvm_properties = DEFAULT_JVM_PROPERTIES,
-        java_home: str = os.environ.get('JAVA_HOME', None),
+        jvm_args = None,
+        jvm_properties = None,
+        java_home = None,
         extra_classpath = [],
         propfile: str = None,
         config = None):
     """ This function uses the default DH property file to embed the Deephaven server and starts a Deephaven Python
     Script session. """
+    if not jvm_args:
+        jvm_args = DEFAULT_JVM_ARGS
```

Comment:
Alternatively, we could use the style `jvm_args = jvm_args or DEFAULT_JVM_ARGS`, thoughts?

Reply:
I think it might be more Pythonic to change the server __init__ to 
```
if jvm_args:
    start_jvm(jvm_args = jvm_args)
else:
   start_jvm()
```

Reply:
Arguably, we _should_ be plumbing through jvm_properties and jvm_home though too - once you have more than one argument, the approach you are suggesting doesn't work...